services/checks/checks.go

```diff
@@ -406,10 +428,9 @@ func (s *Service) processResults(ctx context.Context, check check.Check, target
 	l.Debugf("Results: %+v", results)
 
 	alertsIDs := make([]string, len(results))
-	for i, result := range results {
+	for i, result := range results { //nolint:gosec
```

Comment:
I wonder if we know what is "G601: Implicit memory aliasing in for loop" and if we should disable that particular rule globally.

Reply:
That's because I pass pointer `results` into `createAlert` function, but that function is synchronous and it makes a copy of required `results` fields.